filament/src/materials/skybox.mat

```diff
@@ -33,11 +34,26 @@ fragment {
 #define SHAPR_SKYBOX_TYPE_SOLID_COLOR 0
 #define SHAPR_SKYBOX_TYPE_GRADIENT 1
 #define SHAPR_SKYBOX_TYPE_ENVIRONMENT 2
+#define SHAPR_SKYBOX_TYPE_CHECKERBOARD 3
 
     float3 approxInverseSRGB(float3 srgb) {
         return pow(srgb, vec3(2.2));
     }
 
+    float3 shaprCheckerboardPattern(float2 positionScreenPixelSpace) {
+        const float colorLight = 1.0;
+        const float colorDark = 0.53;
+
+        const float squareSizePixels = 8.0;
+
+        float2 sinParam = positionScreenPixelSpace / squareSizePixels * PI;
+        int2 patterns = int2(saturate(1000.0 * sin(sinParam)));
```

Comment:
can we do this without sin() to spare ALU? e.g. something like https://github.com/mattdesl/glsl-checker

Reply:
I'd also suggest using e.g. `mod` to obtain the checkerboard pattern.

Reply:
Yes, absolutely. I'm on it.

Reply:
thanks